modules/ptp-dual-ports-oc.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/about-ptp.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ptp-dual-ports-oc_{context}"]
+= Using dual-port NICs to improve redundancy for PTP ordinary clocks
+
+{product-title} supports single and dual-port networking interface cards (NIC) as ordinary clocks for PTP timing. To improve redundancy, you can configure a dual-port NIC with one port as active and the other as standby.
+
+:FeatureName: Configuring linuxptp services as an ordinary clock with dual-port NIC redundancy
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+In this configuration, the ports in a dual-port NIC operate as follows:
+
+* The active port functions as an ordinary clock in the `Following` port state.
+
+* The standby port remains in the `Listening` port state.
+
+* If the active port fails, the standby port transitions to active to ensure continued PTP timing synchronization.
+
+If both ports become faulty, the clock state moves to the `HOLDOVER` state, then the `FREERUN` state when the holdover timeout expires, before resyncing to a leader clock.
```

Comment:
```suggestion
* If both ports become faulty before resyncing to a leader clock, the clock state moves to the `HOLDOVER` state, and then to the `FREERUN` state when the holdover timeout expires.
```

Reply:
I think not technically accurate, so leaving as is